### DISCUSSION TITLE
How to Compatible with Lower Version Browsers in Nuxt3？

### DISCUSSION THREAD

**Discussion by QingFull** [nuxt/nuxt#20465] [Questions]
How to Compatible with Lower Version Browsers in Nuxt3？

**Comment by ndragun92** [comment]
[The browser and Node.js support](https://nuxt.com/blog/v3#the-browser-and-nodejs-support)
Nuxt 3 officially supports evergreen browsers only. The "core browser set" is what we (And [web.dev](http://web.dev/) team) believe most developers need to support most of the time in the absence of specific constraints. It takes into account [usage numbers](https://caniuse.com/usage-table), developer expectations, and [existing support in](https://make.wordpress.org/core/handbook/best-practices/browser-support/) [the ecosystem](https://angular.io/guide/browser-support). The core browser set targets the 2 most recent major versions of Chrome, Firefox, and Edge on a monthly basis and Safari on a yearly basis.
On the server side, Nuxt 3 supports Node.js 14, 16, 18, and 19 at the moment. We encourage everyone to use the latest LTS releases of Node.js, we push them once widely adopted by major deployment platforms. This means we keep supporting Node.js versions as long as they are supported by the Node.js team on a rolling basis in non-major releases of Nuxt. Since 14.x is being end-of-life soon, we highly encourage you to update to the latest 18.x whenever possible.
